### PR TITLE
【RESLESE 2.4.2】LICENSE

### PR DESCRIPTION
**Pull request by SaberSola**

### CODE REVIEW

File: shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/LICENSE [hunk 1]
```diff
@@ -361,6 +361,21 @@ The text of each license is the standard Apache 2.0 license.
     zkclient 0.10: https://github.com/sgroschupf/zkclient, Apache 2.0
     zookeeper 3.5.6: https://github.com/apache/zookeeper, Apache 2.0
     zookeeper-jute 3.5.6: https://github.com/apache/zookeeper, Apache 2.0
+    byte-buddy 1.12.6: https://github.com/raphw/byte-buddy, Apache 2.0
+    byte-buddy-agent 1.12.6: https://github.com/raphw/byte-buddy, Apache 2.0
+    jaeger-client 1.6.0 https://github.com/jaegertracing/jaeger Apache 2.0
+    opentracing-api 0.33.0 https://github.com/opentracing/opentracing-java Apache 2.0
```
Comment (tuohai666) [inline shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/LICENSE:367]: opentraccing licenses already exist.

File: shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/NOTICE [hunk 2]
```diff
@@ -305,6 +305,26 @@ This product includes software developed at CoreOS, Inc.
 
 ========================================================================
 
+byte-buddy NOTICE
```
Comment (tuohai666) [inline shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/NOTICE:308]: Please also add zipkin NOTICE.

**Review by tuohai666 — CHANGES_REQUESTED** [review]

File: shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/LICENSE (hunk 1 above)
Comment (SaberSola) [reply shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/LICENSE]: remove done

File: shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/NOTICE (hunk 2 above)
Comment (SaberSola) [reply shenyu-dist/shenyu-bootstrap-dist/src/main/release-docs/NOTICE:308]: add done

**Review by tuohai666 — APPROVED** [review]

**Pull request merged**